Cubical/Data/NatMinusOne/Base.agda

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS --cubical --no-exact-split --safe #-}
+module Cubical.Data.NatMinusOne.Base where
+
+open import Cubical.Core.Primitives
+open import Cubical.Data.Nat
+
+data ℕ₋₁ : Set where
+  neg1 : ℕ₋₁
+  suc  : (n : ℕ₋₁) → ℕ₋₁
+
+_+₋₁_ : ℕ → ℕ₋₁ → ℕ₋₁
+0 +₋₁ n = n
+suc m +₋₁ n = suc (m +₋₁ n)
+
+ℕ→ℕ₋₁ : ℕ → ℕ₋₁
+ℕ→ℕ₋₁ n = n +₋₁ suc neg1
```

Comment:
Changing line this to `ℕ→ℕ₋₁ n = suc (n +₋₁ neg1)` would make the definition of `S n` easier to use. As-is one has to pattern match on `n` in order to do anything with `S n`, but with this change we would have definitionally that `S n ≡ Susp (S₊ (n +₋₁ neg1))` and thus, for example, would have that `north : S n` for all `n`.